tools/acrntrace/README.rst

```diff
@@ -12,6 +12,78 @@ A ``scripts`` directory includes scripts to analyze the trace data.
 Usage
 *****
 
+The ``acrntrace`` tool runs on the Service OS (SOS) to capture trace data and
+output to trace file under ``/tmp/acrntrace`` with raw (binary) data format.
+
+Options:
+
+    -h:
+       print this message
+
+    -t:
+       period_in_ms: specify polling interval [1-999]
+
+    -c:
+       clear the buffered old data
+
+The ``acrntrace_format.py`` is a offline tool for parsing trace data (as output
+by acrntrace) to human-readable formats based on given format.
+
+acrntrace_format.py [options] [formats] [trace_data]
+
+          [options]
+
+          -h: print this message
+
+          Parses trace_data in binary format generated by acrntrace and
+          reformats it according to the rules in the [formats] file.
+          The rules in formats should have the format ({ and } show grouping
+          and are not part of the syntax):
+
+          {event_id}{whitespace}{text format string}
+
+          The textual format string may include format specifiers, such as
```

Comment:
Is this a complete list of format strings?  This could really use a good example to clarify how to write this format file.